src-tauri/src/main.rs

```diff
@@ -66,12 +67,13 @@ async fn farming(path: String, reward_address: String, plot_size: u64) -> bool {
 }
 
 #[tauri::command]
-async fn start_node(path: String) -> String {
-    let node_name = node::generate_node_name();
-    node::init_node(path.into(), node_name.clone())
-        .await
-        .unwrap();
-    node_name
+fn init_node_name() -> String {
```

Comment:
Not a fan that this logic is forwarded through frontend. Don't we have a config on the backend? If not it would be preferred to move node name generation to the frontend completely I think instead of exposing `init_node_name` from the backend.

Reply:
We don't have a config for backend at the moment. I will try to move this function to frontend, thanks for the suggesiton